tests/api.sipnet.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<pecan>
+  <pfts>
+    <pft>
+      <name>temperate.coniferous</name> 
+    </pft>
+  </pfts>
+
+  <meta.analysis>
+    <iter>3000</iter>
+    <random.effects>FALSE</random.effects>
+    <threshold>1.2</threshold>
+    <update>AUTO</update>
+  </meta.analysis>
+
+  <ensemble>
+    <variable>NPP</variable>
+  </ensemble>
+
+  <sensitivity.analysis>
+    <quantiles>
+      <sigma>-1</sigma>
+      <sigma>1</sigma>
+    </quantiles>
+    <variable>NPP</variable>
+  </sensitivity.analysis>
+
+  <model>
+    <type>SIPNET</type>
+    <revision>r136</revision>
```

Comment:
probably do want to use a more up-to-date version of SIPNET

Reply:
Sure...is it the `ssr` revision? Or something else?

Reply:
Good question. I just checked https://github.com/PecanProject/sipnet/releases and didn't realize that we haven't tagged a new release in over a year. For testing purposes I think we can leave at r136 for now, but the master branch has some fixes that folks should be using for science-focused runs.

Reply:
Cool...So I'll leave it that way for now...n once the newer version is tagged and added to the db, we can modify the test file as required.

Reply:
and we need to build a container with that later release.